common/types/string.go

```diff
@@ -22,12 +22,12 @@ import (
 	"strings"
 	"time"
 
+	"github.com/golang/protobuf/ptypes"
+	structpb "github.com/golang/protobuf/ptypes/struct"
```

Comment:
Why move imports to here?

Reply:
I'm trying to separate the imports into three categories. Hopefully my earlier explanation makes sense.